platformio_esp32_envs.ini

```diff
@@ -22,6 +22,8 @@ build_flags               = ${mqtt_flags.build_flags}
                             -DCONFIG_FREERTOS_ASSERT_DISABLE
                             -DCONFIG_LWIP_ESP_GRATUITOUS_ARP
                             -DCONFIG_LWIP_GARP_TMR_INTERVAL=30
+upload_port               = /dev/cu.wchusbserial14630
```

Comment:
Don't set the upload port, especially not in the common section.
For example this port for sure does not exist on my Windows environment.

Also upload speed of 921600 is for sure not advised to be used on all boards.

Reply:
Yes I know, this was an error of mine and I already changed it before you review it.